recipes/cis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: cis
+  version: "1.3.2"
+
+source:
+  #path: ../../cis
+  git_url: https://github.com/cedadev/cis.git
+  git_tag: 1.3.2
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
```

Comment:
Please remove the `build.sh` and `bld.bat` and add the following entry to the `build` field (if using setuptool):

```yaml
script: python setup.py install --single-version-externally-managed --record=record.txt
```

If you are not using setuptools (I don't see it in the dependency list) you should do:

```yaml
script: python setup.py install
```


Do the same for `pyhamcrest`.